docs/sources/k6/next/javascript-api/k6-browser/page/on.md

```diff
@@ -72,3 +67,62 @@ export default async function () {
 ```
 
 {{< /code >}}
+
+### Metric event Example
+
+{{< code >}}
+
+```javascript
+import { browser } from 'k6/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
+  const page = await browser.newPage();
+
+  // Here we call page.on('metric). It's possible to call page.on('metric') more
+  // than once, and the callback function will be executed in the order page.on
+  // was called.
```

Comment:
I'd explain what we're doing first. And then explain the other possible use cases for clarity.

```suggestion
  // We first register a handler using page.on('metric').
  // Calling page.on('metric') multiple times are allowed. The registered handlers
  // will be executed in the order page.on was called.
```

Reply:
Thanks for the suggestion! I went with:

```suggestion
  // We first register a handler using page.on('metric'). Calling page.on('metric')
  // multiple times is allowed. The registered handlers will be executed in the
  // order page.on was called.
```

f27607252bf72393d0a06e12f74220e71a2dba0e